content/courses/native-onchain-development/serialize-instruction-data-frontend.md

```diff
@@ -269,7 +282,7 @@ submit a movie review and have it stored on Solana’s network. We’ll build th
 app a little bit at a time over the next few lessons, adding new functionality
 each lesson.
 
-![Movie review frontend](/public/assets/courses/unboxed/movie-reviews-frontend.png)
+![Movie review frontend](/public/assets/courses/superteam/movie-review-frontend-dapp.png)
```

Comment:
Just move this to /public/assets/courses/movie-review-frontend-dapp.png

Reply:
fixed ✅ - check #346